<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KomintentiKorekcija</class>
 <widget class="QWidget" name="KomintentiKorekcija">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>500</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#0000ff;&quot;&gt;Коминтенти - Корекција &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Шифра</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="sifra_Edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Назив на коминтент</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="naziv_Edit"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Корегирај</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/sterna/Resources/key_f9.png</normaloff>:/sterna/Resources/key_f9.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Адреса</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="adresa_Edit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="telefon_Edit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Мобилен</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="mobilen_Edit"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Жиро сметка</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="ziro_Edit">
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Единствен даночен број</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="danocen_Edit"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Банка Депонент</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLineEdit" name="banka_Edit"/>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Шифра на дејност</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QLineEdit" name="dejnost_Edit"/>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Матичен број</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLineEdit" name="mb_Edit"/>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>ЕМБГ(физичко лице)</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QLineEdit" name="zab1_Edit">
      <property name="maxLength">
       <number>13</number>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Лична карта (физичко лице)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QLineEdit" name="zab2_Edit"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Рабат</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QLineEdit" name="rabat_Edit"/>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Град (населено место)</string>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QLineEdit" name="grad_Edit"/>
    </item>
    <item row="15" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sifra_Edit</tabstop>
  <tabstop>naziv_Edit</tabstop>
  <tabstop>adresa_Edit</tabstop>
  <tabstop>telefon_Edit</tabstop>
  <tabstop>mobilen_Edit</tabstop>
  <tabstop>ziro_Edit</tabstop>
  <tabstop>danocen_Edit</tabstop>
  <tabstop>banka_Edit</tabstop>
  <tabstop>dejnost_Edit</tabstop>
  <tabstop>mb_Edit</tabstop>
  <tabstop>zab1_Edit</tabstop>
  <tabstop>zab2_Edit</tabstop>
  <tabstop>rabat_Edit</tabstop>
  <tabstop>grad_Edit</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
